artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/JournalStorageManager.java

```diff
@@ -369,12 +369,11 @@ protected SequentialFile createFileForLargeMessage(final long messageID, final b
       LargeMessagePersister.getInstance().decode(buff, largeMessage);
 
       if (largeMessage.containsProperty(Message.HDR_ORIG_MESSAGE_ID)) {
-         // for compatibility: couple with old behaviour, copying the old file to avoid message loss
-         long originalMessageID = largeMessage.getLongProperty(Message.HDR_ORIG_MESSAGE_ID);
```

Comment:
Hi, I wonder in which case NumberFormatExc would be thrown because of the fact that message has already contained property HDR_ORIG_MESSAGE_ID.

Reply:
i answered in jira, or do you prefer to discuss here (short form, reason for jms id in orig core property is unclear)

Reply:
Problem is that you can set the Core-message-ID and feed it with a non-number. This then fails when reading it on journal initialization.
The format "ID:<UUID>" is used as JMS-message-id (in contrast to core-message-id), so this seems to end-up sometimes as core-message-id as well.

In any case, I would like to request the retrieval to be moved inside the compatibility block, since it is just unneeded outside.

Reply:
test?

Reply:
Any request to change the commit or other considerations?